core/tests/protractor/embedding.js

```diff
@@ -66,8 +68,8 @@ describe('Embedding', function() {
       'Exploration completed'
     ];
 
-    users.createUser('user1@embedding.com', 'user1Embedding');
-    users.login('user1@embedding.com', true);
+    usersPage.createUser('user1@embedding.com', 'user1Embedding');
```

Comment:
Same, although an array shouldn't be necessary. Maybe just:

```javascript
var USER1 = {name: 'user1Embedding', address: 'user1@embedding.com'};
var USER2 = {name: 'user2Embedding', address: 'user2@embedding.com'};
```

Reply:
okay